### ISSUE TITLE
Message and File columns shall be before Code column

### ISSUE THREAD

**Issue by sandy081** [microsoft/vscode#150792]
Testing #150750

![image](https://user-images.githubusercontent.com/10746682/171136423-eab98f9e-6b04-46b6-a681-1e2bc6d26b54.png)

IMO, message and file info is more important than code

**Comment by lszomoru** [comment]
Ideally we would want to look at adding support for reordering columns to the table widget.

**Comment by sandy081** [comment]
Agreed that reordering helps, but the defaults shall be good. If users think code is important then they can reorder to the front.
